build_requirements.py

```diff
@@ -0,0 +1,98 @@
+import sys
+
+sys.path.append('./')
+from custom_conf import *
+
+# The file contains helper functions and the mechanism to build the
+# .sphinx/requirements.txt file that is needed to setup the virtual
```

Comment:
```suggestion
# .sphinx/requirements.txt file that is needed to set up the virtual
```
Sorry, nitpicking. ;)

Reply:
Praise: No need to be sorry, this is very welcome :)

Reply:
Note: Resolved with c860cca8def2837cf743eb8548039cf0ba5e5e0f